CONTRIBUTING.md

```diff
@@ -170,6 +170,16 @@ for coding advice). The code must adhere to the following robustness principles
 are important for software that runs autonomously and continuously without direct
 interaction with a human (such as this Collector).
 
+### Recommended Libraries / Defaults
+
+In order to simplifiy developing within the project, library recommendations have been set
+and should be followed.
+
+Scenario | Recomended                                   | Rationale 
+----------------------------------------------------------------------------
+Hashing  | ["hashing/fnv"](https://pkg.go.dev/hash/fnv) | The project adopted this as the default hashing method due to the efficency and is reasonable for non cryptographic use
```

Comment:
Why is this not in the Library Recommendations section? Is this somehow special?

Reply:
The added recommended lib sections that I had added was specific to testing but I am happy to shift it into one spot if it makes more sense :)